rmw_zenoh_cpp/src/qos.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
```

Comment:
Should we put this in the `impl` directory instead? I purposely put all the non-RMW stuff in there

Reply:
Good point. Done in 39e29d7.

Reply:
If you want, we can also move the qos.hpp to the implementation detail folder, since it can work as a private header. Unsure on that though.

Reply:
Good catch. Done in 73bf424.